cmd/minishift/cmd/delete.go

```diff
@@ -68,6 +68,7 @@ func runDelete(cmd *cobra.Command, args []string) {
 	}
 
 	removeInstanceConfigs()
+	removeProfileFromConfig()
```

Comment:
Or as we discussed a `minishift delete --all` could work as well. Either way, once needs a way to delete everything, not only the machines directory, but also cache, add-ons and config.